exchange/exchange.go

```diff
@@ -269,7 +269,7 @@ func (e *exchange) HoldAuction(ctx context.Context, r AuctionRequest, debugLog *
 	gdprDefaultValue := e.parseGDPRDefaultValue(r.BidRequestWrapper.BidRequest)
 
 	// Slice of BidRequests, each a copy of the original cleaned to only contain bidder data for the named bidder
-	bidderRequests, privacyLabels, errs := cleanOpenRTBRequests(ctx, r, requestExt, e.bidderToSyncerKey, e.me, gdprDefaultValue, e.privacyConfig, e.gdprPermsBuilder, e.tcf2ConfigBuilder, e.hostSChainNode)
+	bidderRequests, privacyLabels, errs := cleanOpenRTBRequests(ctx, r, requestExt, e.bidderToSyncerKey, e.me, gdprDefaultValue, e.privacyConfig, e.gdprPermsBuilder, e.tcf2ConfigBuilder, e.hostSChainNode, e.bidderInfo)
```

Comment:
A refactor sounds great and I like this idea, but not on the Exchange object. That's too big. Let's break it out into it's own thing if we go this approach.

Reply:
That goes along with Scott's comment above. This could definitely use a refactor. Perhaps even make it a method of the exchange, so it has direct access to the exchange's fields rather than needing to be passed them explicitly. It already had 6 fields from the exchange struct. My addition bumps it up to 7. That may not be too bad of a refactor for this small PR.